woocommerce-myparcel.php

```diff
@@ -113,6 +113,14 @@ private function boot(): void
                 deactivate_plugins(plugin_basename(__FILE__));
             });
         }
+
+        $actions = get_option(Pdk::get('myParcelActions'), []);
+        array_map(static function ($callback, $hook) use ($actions) {
```

Comment:
in dit stuk mist coverage, en het is een beetje slordig zo hier. gaarne in een aparte functie zetten 🙏🏼